DatePicker/DatePicker.js

```diff
@@ -161,7 +161,7 @@ const DatePicker = Pure(
  * @function
  * @memberof sandstone/DatePicker
  * @param {Date} date `Date` to convert
- * @returns {String?} Converted date or `null` if `date` is invalid
+ * @returns {String|Null} Converted date or `null` if `date` is invalid
```

Comment:
How will this change affect the resulting doc? If you change the doc, please provide a link to the new document.